warp/Network/Wai/Handler/Warp/File.hs

```diff
@@ -34,12 +34,17 @@ data RspFileInfo = WithoutBody H.Status
 ----------------------------------------------------------------
 
 conditionalRequest :: I.FileInfo
-                   -> H.ResponseHeaders -> IndexedHeader
+                   -> H.ResponseHeaders
+                   -> IndexedHeader -- Response
+                   -> IndexedHeader -- Request
```

Comment:
`-- ^ Request` haddock comments might be better.

Reply:
Done